StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkURLGenerator.swift

```diff
@@ -95,7 +95,9 @@ class LinkURLGenerator {
 
 extension LinkURLParams {
     func toURLEncodedBase64() throws -> String {
-        let encodedData = try JSONEncoder().encode(self)
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys]
+        let encodedData = try encoder.encode(self)
```

Comment:
I'm guessing this gives us repeatability for the link that is being generated?

Reply:
Yeah, the order of the JSON is different on macOS vs iOS